.github/workflows/ci.yml

```diff
@@ -71,6 +71,8 @@ jobs:
         run: ./scripts/test/stub-data.sh
       - name: Tests
         run: |
+          touch frontends/mit-open/build/index.html
```

Comment:
_added this `touch` for discussion._

One issue I ran into when serving the webpack build in `handle_40x` handlers without using a Django template is that the file won't exist without running the webpack build. And apparently `test_initials_avatar_fake_user`, triggers handle404 (though eventually redirects to `"static/images/avatar_default.png"`). So the tests fail without something at `frontends/mit-open/build/index.html`.

Not sure the best way to resolve this. `touch` doesn't seem right.